### PR TITLE
initial commit of home page

### PR DESCRIPTION
**Pull request by Arpit-dotcom**
created homepage for E-commerce
image
image

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.img-size{
+    margin: 0;
+    object-fit: cover;
+    height: auto;
+}
+
+.main-footer{
+    background-color: #7f1d1d;
```
Comment (Rohitprasad83) [inline homepage.css:45]: background-color is used more than once, consider changing it to a variable

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Shopzilla | Home</title>
+    <link
+      rel="stylesheet"
+      href="https://web-beauty.netlify.app/css/web-beauty.css"
+    />
+    <link rel="stylesheet" href="homepage.css" />
+    <link rel="stylesheet" href="/font-awesome/css/all.css">
+    <!-- <script src="https://kit.fontawesome.com/dcbd873a92.js" crossorigin="anonymous"></script> -->
```
Comment (Rohitprasad83) [inline homepage.html:14]: remove comment codes  if not required.

**Review by Rohitprasad83 — APPROVED** [review]
LGTM!

**Review by rohanmathur91 — APPROVED** [review]
LGTM

**Pull request merged**